frontend/pages/SoftwarePage/SoftwareTitles/SoftwareTableConfig.tsx

```diff
@@ -213,7 +214,12 @@ const generateTableHeaders = (
           <LinkCell
             path={PATHS.SOFTWARE_DETAILS(id.toString())}
             customOnClick={onClickSoftware}
-            value={name}
+            value={
+              <>
+                <SoftwareIcon name={name} source={source} />
+                {name}
```

Comment:
does this render with the correct spacing between the name and the icon? if not we can wrap the name with a span and add some margin.